src/hct_mis_api/api/endpoints/rdi/push_people.py

```diff
@@ -186,6 +197,9 @@ def _create_document(self, member: PendingIndividual, doc: Dict) -> None:
             program=member.program,
         )
 
+    def _create_account(self, member: PendingIndividual, account: Dict) -> None:
+        PendingAccount.objects.create(individual=member, **account)
```

Comment:
we should pop number, is_unique and unique_key in a way and put the rest into data field